rootfs/Dockerfile

```diff
@@ -3,13 +3,14 @@ FROM gcr.io/google_containers/ubuntu-slim:0.4
 ENV TERM=xterm
 
 # disable source repos (speeds up apt-get update)
-RUN sed -i -e 's/^deb-src/#deb-src/' /etc/apt/sources.list && \
-apt-get update && \
-apt-get install -y \
-	bash \
-	curl \
-	net-tools \
-	procps && \
-	apt-get clean && \
-	rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* /usr/share/man /usr/share/doc && \
-bash -c "mkdir -p /usr/share/man/man{1..8}"
+RUN sed -i -e 's/^deb-src/#deb-src/' /etc/apt/sources.list \
+	&& apt-get update \
+	&& apt-get upgrade -y \
```

Comment:
Using `upgrade` with some additional things I can PR later on gives me this

```
quay.io/helgi/base   latest              e52919f03ebf        42 seconds ago      74.41 MB
```

`dist-upgrade` installed all those other not so needed packages

```
quay.io/helgi/base   latest              e5e1094240e7        12 seconds ago      95.88 MB
```

So pretty in sync with the difference @mboersma showed